src/views/Dashboard.jsx

```diff
@@ -0,0 +1,309 @@
+                src="https://res.cloudinary.com/yilin1234/image/upload/v1692498189/f__2_-removebg-preview_xxhyv5.png"
+                alt="logo"
+              />
+            </Link>
+          </div>
+          <nav className="flex flex-1 flex-col">
+            <ul className="flex flex-1 flex-col gap-y-7">
+              <li>
+                <ul className="-mx-2 space-y-1">
+                  {navigation.map((item) => (
+                    <li
+                      key={item.name}
+                      onClick={() => setCurrentTab(item.name)}
+                    >
+                      <Link
+                        to={item.href}
+                        className={classNames(
+                          item.current
+                            ? "bg-gray-50 text-indigo-600"
+                            : "text-gray-700 hover:text-indigo-600 hover:bg-gray-50",
+                          "group flex gap-x-3 rounded-md p-2 text-sm leading-6 font-semibold"
+                        )}
+                      >
+                        <item.icon
+                          className={classNames(
+                            item.current
+                              ? "text-indigo-600"
+                              : "text-gray-400 group-hover:text-indigo-600",
+                            "h-6 w-6 shrink-0"
+                          )}
+                          aria-hidden="true"
+                        />
+                        {item.name}
+                      </Link>
+                    </li>
+                  ))}
+                </ul>
+              </li>
+            </ul>
+          </nav>
+        </div>
+      </div>
+
+      <div className="lg:pl-72">
+        <div className="sticky top-0 z-40 flex h-16 shrink-0 items-center gap-x-4 border-b border-gray-200 bg-white px-4 shadow-sm sm:gap-x-6 sm:px-6 lg:px-8">
+          <button
+            type="button"
+            className="-m-2.5 p-2.5 text-gray-700 lg:hidden"
+            onClick={() => setSidebarOpen(true)}
+          >
+            <Bars3Icon className="h-6 w-6" aria-hidden="true" />
+          </button>
+
+          {/* Separator */}
+          <div className="h-6 w-px bg-gray-200 lg:hidden" aria-hidden="true" />
+
+          <div className="flex flex-1 gap-x-4 self-stretch lg:gap-x-6">
+            <div className="relative flex flex-1"></div>
+            <div className="flex items-center gap-x-4 lg:gap-x-6">
+              {/* Separator */}
+              <div
+                className="hidden lg:block lg:h-6 lg:w-px lg:bg-gray-200"
+                aria-hidden="true"
+              />
+
+              {/* Profile dropdown */}
+              <Menu as="div" className="relative">
+                <Menu.Button className="-m-1.5 flex items-center p-1.5">
+                  <img
+                    className="h-10 w-10 rounded-full bg-gray-50 object-cover"
+                    src={currentUser ? currentUser.photoURL : "loading"}
```

Comment:
instead of using the string "loading" as a fallback for the image source, we can use a default or placeholder image. This way we can prevent broken image icons from appearing if `currentUser.photoURL` is null or undefined.